src/legacy/core_plugins/kibana/public/visualize/embeddable/visualize_embeddable.ts

```diff
@@ -57,6 +57,10 @@ export interface VisSavedObject extends SavedObject {
   title: string;
   uiStateJSON?: string;
   destroy: () => void;
+  savedSearchRefName?: string;
+  savedSearchId?: string;
+  savedSearch?: any;
```

Comment:
Isn't this of `SavedQuery` type

Reply:
It's https://github.com/elastic/kibana/blob/85aea35c944abf3bb8d315cb599d0e19ce58ed1f/src/legacy/core_plugins/kibana/public/discover/types.d.ts#L24
thanks for the hint, one `any` less